model/src/test/kotlin/AbstractDependencyNavigatorTest.kt

```diff
@@ -85,75 +87,88 @@ abstract class AbstractDependencyNavigatorTest : WordSpec() {
             }
         }
 
-        "scopeDependencies" should {
-            "return a map with scopes and their dependencies for a project" {
-                val scopeDependencies = navigator.scopeDependencies(testProject)
-
-                scopeDependencies.keys should containExactlyInAnyOrder("compile", "test")
-
-                scopeDependencies["compile"] shouldNotBeNull {
-                    this should haveSize(17)
-                    this should containAll(
-                        Identifier("Maven:com.typesafe.akka:akka-actor_2.12:2.5.6"),
-                        Identifier("Maven:org.scala-lang:scala-reflect:2.12.2"),
-                        Identifier("Maven:org.scala-lang:scala-library:2.12.15")
-                    )
-                }
+        "projectDependencies" should {
+            "return all dependencies from all scopes for a project" {
+                val scopeDependencies = navigator.projectDependencies(testProject)
 
-                scopeDependencies["test"] shouldNotBeNull {
-                    this should haveSize(6)
-                    this should containAll(
-                        Identifier("Maven:org.scalacheck:scalacheck_2.12:1.13.5"),
-                        Identifier("Maven:org.scalactic:scalactic_2.12:3.0.4")
-                    )
-                }
+                scopeDependencies should containAll(
```

Comment:
Because not *all* IDs are expected to be contained. This matches the original code which was also using `containAll` per scope for a few exemplary IDs, plus `haveSize` assertions for the real number of IDs.

Reply:
I assumed this should be all packages because you dropped the `haveSize` check. If not, I think you should reintroduce it to match the test name.